src/utils/__tests__/performanceTestSuite.test.ts

```
@@ -0,0 +1,235 @@
+/**
+ * Performance Test Suite Tests
+ * Unit tests for the performance testing system
+ */
+
+import { it } from 'node:test';
+import { describe } from 'yargs';
+import { it } from 'node:test';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'it'. Consider to delete it or rename it.

'it' is defined here:


(lint/suspicious/noRedeclare)